src/main/groovy/org/shipkit/internal/gradle/downstream/test/RunDownstreamTestTask.java

```diff
@@ -15,9 +15,9 @@
 /**
  * This task run external process and additionally store output of external process to file.
  */
-public class RunTestReleaseTask extends DefaultTask {
+public class RunDownstreamTestTask extends DefaultTask {
```

Comment:
It seems that the name of this task is too specific. The task only runs an arbitrary command, and is not coupled with downstream testing. The only thing that couples the implementation of this task with downstream testing is the repo name and log message emitted before the task runs, which can be managed differently, e.g.

How about calling this task 'SilentExecTask'?